tests/com/twitter/intellij/pants/integration/OSSPantsCompileActionsTest.java

```diff
@@ -77,6 +86,36 @@ public void testCompileAllTargetsInModuleAction() throws Throwable {
     assertFalse(compileAllTargetsInModuleAction.doCleanAll());
   }
 
+  public void testCompileTargetsInSelectedEditor() throws Throwable {
+    doImport("examples/tests/scala/org/pantsbuild/example");
```

Comment:
there are two tests classes and their associated targets, so when the 2nd one is opened, there will be 2 files opened in the editor, but one is selected. I can add a comment to it.